### ISSUE TITLE
V13-beta Templates not rendering variables in docker images

### ISSUE THREAD

**Issue by senner** [frappe/frappe_docker#387]
Using the v13-beta docker images and for some reason on multiple different sites and templates like job_application or kb article, even new generated sites jinja template variables are not replaced with their values such as below. No errors are generated in console or error logs, and nothing about them were customized.

/job_application?new=1&job_title=JobTitle

<link type="text/css" rel="stylesheet" href="/assets/css/erpnext-web.css?ver={{ build_version }}">

or
/kb/pbx/somekbarticlename

sheet" href="/assets/css/frappe-web-b4.css?ver={{ build_version }}">

Same as on all-products page as well

**Comment by revant** [comment]
Can you try restarting the containers?

It copies static assets on start of nginx and gunicorn container. It should work after the assets are copied.

**Comment by senner** [comment]
Yes , we've tried that a stop and start. The asset is 'there' as most other pages work and reference the .css file ver=0.x.x.x.x. just fine with the build number variable processed with the build version in the template. The reference in the template to the asset is what's not replaced.

**Comment by revant** [comment]
I tried latest develop branch on https://labs.play-with-docker.com/ it seem to load fields for `/job_application?new=1&job_title=JobTitle`

**Comment by senner** [comment]
Well this is the v13-beta docker image, just moved to the develop tagged image and we get the same things. We ran 'bench clear-website-cache' and 'bench clear-cache' and same results.  Is there a setting that controls generation of these pages ?  

for now we just added:

/desk#Form/Website Settings under HTML HEADER, ROBOTS and Redirects added :
```
<link type="text/css" rel="stylesheet" href="/assets/css/frappe-web-b4.css?ver=1607528273.0">
		<link type="text/css" rel="stylesheet" href="/assets/css/erpnext-web.css?ver=1607528273.0">
```

**Comment by revant** [comment]
Does it work as part of normal non docker bench?

**Comment by revant** [comment]
I can confirm there is issue with job application form even in non docker bench.

- Create fresh site and complete wizard
- Create and publish `Job Opening`
- As logged in user visit the job application page, (user is possibly `System Manager` role as we just completed the wizard).
- fill up the page, attach file and save.
- It works.

Logout and try as `Guest`, it only takes the form data, It doesn't allow guest to attach file to jobs.

Problem and discussion:

- Job applicants are guests and need to attach their profile as file.
- We can't ask job applicants to create user account and then apply for job.
- Then there will be many unused user accounts created in system.
